infra/cifuzz/actions/run_fuzzers/run_fuzzers_entrypoint.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs specific OSS-Fuzz project's fuzzers for CI tools."""
+import logging
+import os
+import sys
+
+# pylint: disable=wrong-import-position
+# pylint: disable=import-error
+sys.path.append(os.path.join(os.environ['OSS_FUZZ_ROOT'], 'infra', 'cifuzz'))
+import cifuzz
+
+# TODO: Turn default logging to INFO when CIFuzz is stable
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.DEBUG)
+
+
+def main():
+  """Runs OSS-Fuzz project's fuzzers for CI tools.
+  This is the entrypoint for the run_fuzzers github action.
+  This action can be added to any OSS-Fuzz project's workflow that uses Github.
+
+  NOTE: libfuzzer binaries must be located in the ${GITHUB_WORKSPACE}/out
+  directory in order for this action to be used. This action will only fuzz the
+  binary's that are located in that directory. It is reccomended that you add
+  the build_fuzzers action preceding this one.
+
+  NOTE: Any crash report will be in the filepath:
+  ${GITHUB_WORKSPACE}/out/testcase
+  This can be used in parallel with the upload-artifact action to surface the
+  logs.
+
+  Required environment variables:
+    FUZZ_SECONDS: The length of time in seconds that fuzzers are to be run.
+    GITHUB_WORKSPACE: The shared volume directory where input artifacts are.
+    DRY_RUN: If true, no failures will surface.
+
+  Returns:
+    0 on success or 1 on Failure.
+  """
+  fuzz_seconds = int(os.environ.get('FUZZ_SECONDS', 360))
```

Comment:
Let's do 10 minutes by default. 6 seems pretty small and having a non-multiple of 5 bothers me :-)

Reply:
don't forget to change it in the yaml file too :)

Reply:
Done.